index.php

```diff
@@ -0,0 +1,57 @@
+<?php
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link href="https://fonts.googleapis.com/css?family=Cantarell:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/index.css">
+    <title>Pokédex</title>
+</head>
+<body>
+    <main>
+        <form action="index.php" class="nameSearch">
+            <input type="search" name="nameSearch" placeholder="Search by name">
```

Comment:
Minor point, but the get variable I used in the branch that does the legwork is just called "search". You might want to rename it to that to make the mergening easier

Reply:
Changed in f7c4057.